aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -908,14 +974,22 @@ mod tests {
 
         let mut base_update = BTreeMap::new();
         let mut additional_update = BTreeMap::new();
-        base_update.insert(key.clone(), GroupWrite {
-            metadata_op: write_op_with_metadata(0, 100), // create
-            inner_ops: BTreeMap::new(),
-        });
-        additional_update.insert(key.clone(), GroupWrite {
-            metadata_op: write_op_with_metadata(2, 200), // delete
-            inner_ops: BTreeMap::new(),
-        });
+        base_update.insert(
+            key.clone(),
+            GroupWrite::new(
+                write_op_with_metadata(0, 100), // create
```

Comment:
nit from Rati's PR: we can use const `CREATE: u64 = 0` to make type indices in tests more readable

Reply:
Done.